src/object.c

```diff
@@ -733,11 +733,11 @@ robj *getDecodedObject(robj *o) {
  * use ll2string() to get a string representation of the numbers on the stack
  * and compare the strings, it's much faster than calling getDecodedObject().
  *
- * Important note: when REDIS_COMPARE_BINARY is used a binary-safe comparison
+ * Important note: when VALKEY_COMPARE_BINARY is used a binary-safe comparison
```

Comment:
```suggestion
 * Important note: when STRING_COMPARE_BINARY is used a binary-safe comparison
```

VALKEY seems out of place here, let's update it to reference that it is for strings. make sure to update the other defines as well.